setup.py

```diff
@@ -0,0 +1,2 @@
+from setuptools import setup
+setup()
```

Comment:
I prefer to put a blank line at the end of each file to avoid the scary red circle thingy in the diff.